InfraSIM/infrasim-compute
# HWSIM-890, to solve ConfigParser library issue. Switch to library cfgparse.

Test below pass:
1. run "pip install cfgparse".
2. install new source code with the new library, run "sudo python setup.py install".
3. restart network.
4. run "sudo infrasim-vm restart".
5. check vm status.
